docs/technical_development_guide/source/resource_files/sdr_minimal_buildstock.rst

```diff
@@ -0,0 +1,44 @@
+.. _sdr_minimal_buildstock:
+
+SDR Minimal BuildStock
+=======================
+
+The ``sdr_minimal_buildstock.csv`` is a buildstock.csv file that can be used to run test simulations. It contains a minimal set of buildings which together hit all the apply logic blocks in the sdr project yaml file (``project_national/sdr_upgrades_tmy3.yml``). This helps test all the option application by using smallest set of buildings.
+
+File Location
+-------------
+
+The file is located at ``project_national/resources/sdr_minimal_buildstock.csv`` in the ResStock repository.
+
+File Structure
+--------------
+
+The file is structured as a CSV (comma-separated values) file and follows the same format as the buildstock.csv generated by the sampling script. As such, this file can be directly used by the precomputed sampler. In the file:
+
+- Each **row** represents a unique building in the stock
+- Each **column** represents a housing characteristic parameter
+- The first column typically contains a building identifier (Building ID)
+- All subsequent columns contain the specific option values assigned to that building for each parameter
+
+The file contains hundreds of columns, representing all the parameters that define a building in ResStock, including:
+
+- Location parameters (Climate Zone, County, State, etc.)
+- Building physical characteristics (Geometry, Insulation levels, etc.)
+- Equipment characteristics (HVAC systems, Water Heaters, etc.)
+- Occupancy and usage parameters
+
+Example
+-------
+
+Here's a simplified example of how the data appears in the file (showing only a few columns for clarity):
+
+.. code-block:: text
+
+   Building,ASHRAE IECC Climate Zone 2004,County and PUMA,Vintage,Heating Fuel,HVAC Heating Type,...
+   262933,4A,"G3400050, G34002001",<1940,Fuel Oil,Non-Ducted Heating,...
+   306173,4A,"G3400150, G34002202",1970s,Fuel Oil,Non-Ducted Heating,...
+   307627,7A,"G2700170, G27000300",1990s,Propane,Ducted Heating,...
+
+Creation
+--------
+minimal_buildstock.csv file is automatically created by ResStock CI job and committed to project_national/resources directory. The CI uses upgrades_analyzer tool in buildstock-query with the ``project_national/sdr_upgrades_tmy3.yml`` and a fresh 550K sample as input to generate this file. Whenever the yaml apply or the housing characteristics changes, the CI job will regenerate this file and commit to the repo.
```

Comment:
This is a little tough to read, I think we need to add some articles - like "file is automatically created by **the** ResStock CI job. As well as "The CI uses **the** upgrades_analyzer tool, etc.